openstudiocore/src/osversion/VersionTranslator.cpp

```diff
@@ -4099,6 +4099,51 @@ std::string VersionTranslator::update_2_6_1_to_2_6_2(const IdfFile& idf_2_6_1, c
   IdfFile targetIdf(idd_2_6_2.iddFile());
   ss << targetIdf.versionObject().get();
 
+  // We want to enforce that each rendering color has only 0 or 1 parent
+  // Use a set to check if we have already encountered this rendering color
+  std::set<std::string> renderingColorHandlesSet;
```

Comment:
@jmarrec with an IDD change this version translation is required but also more straightforward